### DISCUSSION TITLE
How to get image and pdf preview using ueberzug?

### DISCUSSION THREAD

**Discussion by semi710** [junegunn/fzf#2851] [Q&A]
I cannot figure out how I can have an image and pdf preview using ueberzug.

I have configured pistol and it works fine but not for image and pdf for the meantime I have got a wrapper script that does the job.

Script : [fzf-ueberzogen.sh](http://ix.io/409d)
Helper file : [imgprev](http://ix.io/409e) (pointed by a reddit user.)

using this script gives me the preview with image and pdf and uses pistol for other files.

I am unable to use this with the ctrl-t option of fzf.

**Comment by gmou3** [comment]
Please have a look at #4000 for an alternative script (the provided links don't seem to work anymore).

As for previewing with CTRL-T, you need to `export FZF_CTRL_T_OPTS='--preview "script {}"'`.
